### ISSUE TITLE
PlotSignal ignores yOffset when colorByDensity is used

### ISSUE THREAD

**Issue by swharden** [swharden/ScottPlot#194]
This bug was introduced in #192 and only affects signal plots using the `colorByDensity` argument.

This code should produce non-overlapping graphs, but currently it does not:
```cs
Color[] colors = new Color[]
{
	ColorTranslator.FromHtml("#73D055"),
	ColorTranslator.FromHtml("#1F968B"),
	ColorTranslator.FromHtml("#39568C"),
	ColorTranslator.FromHtml("#440154")
};
plt.PlotSignal(data, color: Color.Red);
plt.PlotSignal(data, yOffset: -40, colorByDensity: colors);
```

**Comment by StendProg** [comment]
#195 add missed `yOffset` param.
I rememder i dragged this `+ yOffset` lots of  time in code. Some time i missed that...

**Comment by swharden** [comment]
No worries! I just merged #195 in. Thank you @StendProg for all of your hard work
